### ISSUE TITLE
[Feature]: don't automatically stack "you can trade this for the next X hours" loot

### ISSUE THREAD

**Issue by emptyrivers** [Cidan/BetterBags#346]
### Describe the feature

I somehow managed to loot 2 irideus fragments today. Had to disable unstackable merge before I could trade the second one to a party member :<

### Describe the implementation

Add an option possibly in here to disable stacking on temporarily tradeable gear, even if the "Merge Unstackable" option is ticked.
![image](https://github.com/Cidan/BetterBags/assets/24878501/1d96389d-2fb5-4aca-aac5-2cd868794d10)

**Comment by Cidan** [comment]
yeah this is coming as soon as we get solid tooltip scanning in place, which is... soon.

**Comment by emptyrivers** [comment]
ah, a 22nd century feature, no worries 😄
